appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">dolphin-emu.desktop</id>
```

Comment:
this should be the app-id of the application.

Reply:
As of right now, the application compiled is `dolphin-emu`. Changing this would actually break the flatpak, to some extent.

Reply:
no. this is unrelated to the compiled application, it's all about the app-id.